src/plugins/unified_histogram/public/chart/chart.tsx

```diff
@@ -69,7 +73,8 @@ export interface ChartProps {
   disableTriggers?: LensEmbeddableInput['disableTriggers'];
   disabledActions?: LensEmbeddableInput['disabledActions'];
   input$?: UnifiedHistogramInput$;
-  lensTablesAdapter?: Record<string, Datatable>;
+  lensAdapters?: UnifiedHistogramChartLoadEvent['adapters'];
```

Comment:
ℹ️ Instead of sending only the tables we know send all the adapters object to Lens flyout. With this way we can fetch correctly the active data. (Solves a bug currently in main more visible on this PR, check first bullet [here](https://github.com/elastic/kibana/pull/162389#discussion_r1322645626)